src/main/java/com/proofpoint/discovery/ProxyStore.java

```diff
@@ -44,10 +44,10 @@
 import java.util.concurrent.atomic.AtomicBoolean;
 import java.util.stream.Stream;
 
-import static com.google.common.base.Throwables.propagate;
 import static com.proofpoint.concurrent.Threads.daemonThreadsNamed;
 import static com.proofpoint.discovery.client.announce.DiscoveryAnnouncementClient.DEFAULT_DELAY;
 import static com.proofpoint.json.JsonCodec.jsonCodec;
+import static org.weakref.jmx.internal.guava.base.Throwables.propagateIfPossible;
```

Comment:
That looks like the wrong import.

Reply:
`ExecutionException` is a checked exception, so the call isn't even necessary.